charts/kyverno/Chart.yaml

```diff
@@ -27,4 +27,4 @@ annotations:
   # valid kinds are: added, changed, deprecated, removed, fixed and security
   artifacthub.io/changes: |
     - kind: added
-      description: Added possibility to define additional init and sidecar container
+      description: Added possibility to define custom image registries
```

Comment:
The old entry is still valid , so this PR would need a new entry.

Reply:
Added this one :)